root/static/scripts/guess-case/modes.js

```diff
@@ -8,7 +8,7 @@ const {assign} = require('lodash');
 
 const {l} = require('../common/i18n');
 const getBooleanCookie = require('../common/utility/getBooleanCookie');
-const {isPrepBracketWord, isPrepBracketSingleWord} = require('./utils');
+const {isPrepBracketWord, isPrepBracketSingleWord, turkicUpperCase, turkicLowerCase} = require('./utils');
```

Comment:
The word _Turkic_ doesn’t make any sense here. See https://www.quora.com/Whats-the-difference-between-Turkic-and-Turkish